lua/lvim/plugins.lua

```diff
@@ -281,6 +281,10 @@ local core_plugins = {
     end,
     disable = lvim.colorscheme ~= "onedarker",
   },
+
+  {
+    "lunarvim/bigfile.nvim",
```

Comment:
should we make this configurable?

@abzcoding

Reply:
I mean, I always do appreciate more customization possibilities :)

But we can do it in a later PR

Reply:
done